openbook_auth/models.py

```diff
@@ -1733,6 +1741,24 @@ def search_followers_with_query(self, query):
 
         return User.objects.filter(followers_query).distinct()
 
+    def get_user_subscriptions(self, max_id=None):
```

Comment:
Argghhh this is confusing.

`def get_user_notificactions_subscriptions` perhaps ?